spec/controllers/merchants_items_controller_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::Merchants::ItemsController do
+  describe "GET related items for that merchant" do
+    it "retrieves items related to that merchant" do
+      merchant = FactoryGirl.create(:merchant, id: 1)
+      5.times do
+        FactoryGirl.create(:item, merchant: merchant)
+      end
+      get :index, params: {
```

Comment:
@concach 
See my comment in spec/controllers/merchants_invoices_controller_spec.rb.

Also on line 22 of this file.